diff --git a/plugin/plugins.vim b/plugin/plugins.vim
index e6817be..4f537de 100644
--- a/plugin/plugins.vim
+++ b/plugin/plugins.vim
@@ -61,7 +61,7 @@ autocmd VimEnter * lsp.Setup()
 def! g:VimCompleteOptionsSet(opts: dict<any>)
     completor.SetOptions(opts)
     for [key, newopts] in opts->items()
-        if !getscriptinfo({ name: $'vimcomplete/autoload/vimcomplete/{key}' })->empty()
+        if !getscriptinfo({ name: $'autoload/vimcomplete/{key}' })->empty()
             var o = eval($'{key}.options')
             o->extend(newopts)
         endif
